### PR TITLE
Version 3

### PR DESCRIPTION
**Pull request by ianardee**

### CODE REVIEW

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -2,9 +2,6 @@ name: Linter
 
 on:
   push:
-    paths:
-      - "**.cs"
-      - ".editorconfig"
```
Comment (sebastianMindee) [inline .github/workflows/linter.yml:7]: Does this work without any values under `push`?

**Review by sebastianMindee — CHANGES_REQUESTED** [review]
Couple of issues to fix, but nothing major

File: .github/workflows/linter.yml (hunk 1 above)
Comment (ianardee) [reply .github/workflows/linter.yml:7]: yes

**Review by sebastianMindee — APPROVED** [review]
:tada:

**Pull request merged**